### PR TITLE
Add version to help and log. Add a -v flag.

### PR DESCRIPTION
**Pull request by yuyantingzero**

### CODE REVIEW

**Comment by yuyantingzero** [comment]
I signed it!

File: perfkitbenchmarker/pkb.py [hunk 1]
```diff
@@ -468,9 +470,13 @@ def Main(argv=sys.argv):
       '%s:  %s' %
       (set_name, benchmark_sets.BENCHMARK_SETS[set_name]['message'])
       for set_name in benchmark_sets.BENCHMARK_SETS]
-  sys.modules['__main__'].__doc__ = __doc__ + (
-      '\nBenchmarks (default requirements):\n\t%s' %
-      _GenerateBenchmarkDocumentation())
+  sys.modules['__main__'].__doc__ = (
```
Comment (skschneider) [inline perfkitbenchmarker/pkb.py]: Looks like the point of this string is to be printed on the local machine. Would os.linesep be more appropriate?

Comment (yuyantingzero) [inline perfkitbenchmarker/pkb.py]: AFAIK, Pkb can only run on unix, linux machine, both using \n to separate lines. Any platform you know \n will be broken?

Comment (skschneider) [inline perfkitbenchmarker/pkb.py]: Not currently, though I imagine we may eventually want to get things to work on Windows as well. Considering we already have \n scattered throughout the rest of the code, it's not a blocking issue.

**Comment by skschneider** [comment]
LGTM

**Pull request merged**